packages/wonder-blocks-modal/components/one-pane-dialog/one-pane-dialog.js

```diff
@@ -139,6 +139,7 @@ export default class OnePaneDialog extends React.Component<Props> {
                     title={title}
                     subtitle={(subtitle: string)}
                     titleId={uniqueId}
+                    testId={testId && `${testId}-ModalHeader`}
```

Comment:
Including `-ModalHeader` makes this a bit more awkward to use and embeds an implementation detail in the testing interface.  I think passing `testId` on its own should be fine.

Reply:
Same situation as described before. Added this suffix here to avoid duplication with the testId set in ModalDialog. Maybe `-header` would be a better name?